tailwind.config.js

```diff
@@ -45,6 +45,7 @@ module.exports = {
           light: '#5E8EBD',
           medium: '#173451',
           dark: '#21303F',
+          button: '#295376',
```

Comment:
Should we be adding colours outside the defined pallet, or seek correction from the design team members?

Reply:
I don't know where the pre-defined color pallet comes from. I don't mind adding colors to it. I will raise this to the stand-up tomorrow.